tests/accessor/accessor_default_values.h

```diff
@@ -0,0 +1,149 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  This file provides functions for tests on accessor default values.
+//
+*******************************************************************************/
+#ifndef SYCL_CTS_ACCESSOR_DEFAULT_VALUES_H
+#define SYCL_CTS_ACCESSOR_DEFAULT_VALUES_H
+
+#include "accessor_common.h"
+#include "catch2/catch_test_macros.hpp"
+
+#include <type_traits>
+
+namespace accessor_default_values_test {
+
+constexpr int expected_dims = 1;
+constexpr sycl::target expected_target = sycl::target::device;
+constexpr sycl::access::placeholder expected_placeholder =
+    sycl::access::placeholder::false_t;
+
+/**
+ * @brief Provides functor that lets verify that local_accessor's template
+ *        parameters have expected default values
+ * @tparam T Current data type
+ * @param type_name Current data type string representation
+ */
+template <typename T>
+void test_exception_for_local_acc(const std::string& type_name) {
```

Comment:
```suggestion
template <typename T>
void test_for_local_acc(const std::string& type_name) {
```
Looks like it would be a bit more consistent this way

Reply:
`test_exception_for_local_acc` has been renamed [here](https://github.com/KhronosGroup/SYCL-CTS/pull/335/commits/c040cc7560db878fb467d794a504c6e5e20a0d0e#diff-9ed3eae8f2889bc04dfc0fad247a8fa4e381e1fff4144e0f1a23670d108f6fe9R34)